api/src/db/models/Brands.ts

```diff
@@ -6,6 +6,7 @@ import { IIntegrationDocument } from './definitions/integrations';
 
 export interface IBrandModel extends Model<IBrandDocument> {
   getBrand(_id: string): IBrandDocument;
+  getBrandByCode(code: string): IBrandDocument;
```

Comment:
almost the same as getBrand(). No need for such duplicated code. Add optional code param to getBrand()